napari_tiledb_bioimg/_reader.py

```diff
@@ -24,15 +24,28 @@ def napari_get_reader(path):
         return None
 
     config = None
+    valid = False
+    if os.path.exists(path):
+        valid=True
+    else: 
+        warnings.warn(f"Not a local valid path {path}")
```

Comment:
Remove? We shouldn't print this warning all the time, IMO - path can be any of the valid ones checked below, this will create a lot of noise in the logs/terminal.

Reply:
Indeed will remove it totally. thanks!